libs/src/api/track.js

```diff
@@ -3,6 +3,23 @@ const CreatorNode = require('../services/creatorNode')
 const Utils = require('../utils')
 const retry = require('async-retry')
 
+const TRACK_PROPS = [
+  'owner_id',
+  'title',
+  'length',
+  'cover_art_sizes',
+  'tags',
+  'genre',
+  'mood',
+  'credits_splits',
+  'release_date',
+  'file_type'
+]
+const TRACK_REQUIRED_PROPS = [
+  'owner_id',
+  'title'
+]
```

Comment:
i think we need to update these values. from discprov the fields are:
```
track_metadata_format = {
    "owner_id": None,
    "title": None,
    "route_id": None,
    "length": 0,
    "cover_art": None,
    "cover_art_sizes": None,
    "tags": None,
    "genre": None,
    "mood": None,
    "credits_splits": None,
    "create_date": None,
    "release_date": None,
    "file_type": None,
    "description": None,
    "license": None,
    "isrc": None,
    "iswc": None,
    "track_segments": [],
    "download": {},
    "is_unlisted": False,
    "field_visibility": None
}
```
also for required props i'm not sure how this was determined but creator node requires owner_id and track_segments, which makes more sense anyway. no harm in also requiring title

Reply:
I didn't want to change this from the current way it works in libs